ts/src/kucoin.ts

```diff
@@ -1929,15 +1929,15 @@ export default class kucoin extends Exchange {
         const stopTriggered = this.safeValue (order, 'stopTriggered', false);
         const isActive = this.safeValue (order, 'isActive');
         let status = undefined;
-        if (isActive === true) {
+        if ((isActive === true) || ('orderId' in order)) {
```

Comment:
@ndubel when we place an order, let's say a market buy kucoin returns this response
```
ResponseBody:
 {"code":"200000","data":{"orderId":"6447a177d8b7bc0001760bb5"}} 
```
how can we be sure the order is open in this case?

Reply:
ok, I agree. But now the same order will be marked as `closed`. Maybe we should do something like this?
```
if ((status === undefined) && !('orderId' in order)) {
    status = 'closed';
}
```

Reply:
@ndubel I'm not sure I understand the rational behind that condition, if don't have `orderId` it means we have a "complete" response right? and in that case we should have `isActive` value
```
    iceberg: false,
    visibleSize: '0',
    cancelAfter: 0,
    channel: 'API',
    clientOid: '6ee04a45-bfc6-4791-8767-6ca9f341ccae',
    remark: null,
    tags: null,
    isActive: true, <---- here
    cancelExist: false,
```

Reply:
you tell about `fetchOpenOrders`/`fetchClosedOrders`, in these cases it's all ok now
I tell about `createOrder`, where we have only `orderId`. Now in case of `createOrder` we have `undefined` status that is transformed to `closed` in the end. So all new created orders comes with `closed` status
https://github.com/ccxt/ccxt/blob/f50679b90de5e01e599c4dcdba49abdadf4a89c1/ts/src/kucoin.ts#L1946-L1948

Reply:
@ndubel yeah but in that case I would leave it as `undefined` because there is no way to tell for sure

Reply:
@carlosmiei yes, I agree! What is the best way to leave it `undefined` (and don't break `fetchClosedOrders`) on your opinion?